source/developer/mobile-developer-setup.rst

```diff
@@ -14,6 +14,82 @@ If you run into any issues getting your environment set up, check the Troublesho
 Development Environment Setup
 ---------------------------------
 
+We strongly recommend the use of a computer that supports macOS to develop and build the Mattermost mobile apps as macOS is required to build the iOS app.
+
+Software Requirements
+---------------------------------
+
+Common
+~~~~~~~~~~~~~~~~~~~
+
+Irrespective of which platform you are going to develop and build, you'll need to install some prerequisite software. For macOS, we recommend using `Homebrew <https://brew.sh/>`_ as a package manager.
+
+Install `NodeJS <https://nodejs.org/en/>`_.
```

Comment:
Should we match formatting of existing developer guide? E.g. number the steps, consistent wording for bullet points (using --> use to match `open`, `Homebrew` to `Use Homebrew` to consistently start with a verb) in-line code blocks for commands (which is two backticks `` in RST)

Reply:
Might help to review the formatting in "Preview" mode https://github.com/mattermost/docs/blob/cb9d92df859d38ab9239b01d2b00bba103165599/source/developer/mobile-developer-setup.rst

Reply:
(Also, see the "Testing Environment Setup" for an example